Tests/SwiftSyntaxBuilderTest/StructTests.swift

```diff
@@ -22,7 +22,7 @@ final class StructTests: XCTestCase {
     let nestedStruct = StructDecl(
       structKeyword: .struct.withLeadingTrivia(.docLineComment("/// A nested struct\n")),
       identifier: "NestedStruct",
-      genericParameterClause: GenericParameterClause {
+      genericParameterClause: GenericParameterClause(rightAngleBracket: .rightAngle.withoutTrailingTrivia()) {
```

Comment:
This is a temporary fix needed because of updates for the `where` keyword. I'm going to remove it with adjusting spaces around angle brackets in my next PR

Reply:
I’m not sure how big the changes for the angle brackets are but feel free to include them in this PR.

Reply:
Not big, however I'd like to make a separate PR for them